### DISCUSSION TITLE
Would using aluminium for the build be recommended?

### DISCUSSION THREAD

**Discussion by theshenthaver** [Fabricatable-Machines/humphrey2#4] [General]
Hi Jens, 

Wow, thank you for putting all this hard work into this design and then sharing it with the world. Its a seriously solid design. 

I'm looking at making this from aluminum, similar to the Humphrey 1. Its quite a bit harder to get Phenolic resin Boards in South Africa. What where the main motivators for moving over to Phenolic Resin boards for the design. I'm assuming it has to do with being easier to machine and the POM Pinions wearing over time? 

Best regards
Theshen

**Comment by JensDyvik** [comment]
Hi Theshen

Thank you for the kind words.

You are correct, the phenolic resin cardboard composite boards are easier and faster to machine than aluminum. They are also a bit cheaper. The surface friction is low, so the machined phenolic rack works well with POM pinions. Self milled alu racks give of oxide over time that accelerates pinion wear. I would not recommend adapting the Humphrey2 design for aluminium sheets. I think the machining time will be quite high. 

Aluminium extrusions are so accessible and convenient to work with these days. Its a more boring approach than milling all the parts yourself from one sheet, but I do think that using a fat alu extrusion for the gantry, smaller extrusions for the machine bed frame and store bought steel rack and pinion would be the best approach.
